tests/test_group.py

```diff
@@ -1443,7 +1459,454 @@ def test_delitem_removes_children(store: Store, zarr_format: ZarrFormat) -> None
 
 
 @pytest.mark.parametrize("store", ["memory"], indirect=True)
-def test_group_members_performance(store: MemoryStore) -> None:
+@pytest.mark.parametrize("impl", ["async", "sync"])
```

Comment:
I'm interested to see if this pattern holds up. Running our `sync` function inside pytests IO loop may eventually lead to problems. But let's see.